### PR TITLE
removes wizard mob atmos requirement

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
makes it so you cant cheese wizard academy by removing one wall
#### Changelog

:cl:    
tweak: tweaked wizard mob  
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
good merge conflict fun

**Comment by Hopekz** [comment]
Wizards not dying to atmos is pretty immersion-breaking IMO.
Either have the wizards have a more believable solution (like all wearing the wizard hardsuit) or not have this in place at all.
I'd much rather have the wizard academy not be a thing anymore rather than having a feature that is inconsistent with how the rest of the game functions.

**Comment by overlord12345** [comment]
Pssst theres legit a version of the wiz thats immune to atmos

**Comment by monster860** [comment]
Ahh the wizard academy. As asst Venus went out twice to try to cheese the wizard academy. Both times he died.

**Comment by boodaliboo** [comment]
> Pssst theres legit a version of the wiz thats immune to atmos

i did this when i didnt have access to mapping tools if this pr gets closed oh well

File: code/modules/mob/living/simple_animal/hostile/wizard.dm [hunk 1]
```diff
@@ -18,13 +18,11 @@
 	melee_damage_lower = 5
 	melee_damage_upper = 5
 	attacktext = "punches"
-	attack_sound = 'sound/weapons/punch1.ogg'
+	attack_sound = 'sound/weapons/punch1.ogg' // this is only here so i can recommit this
 	a_intent = INTENT_HARM
-	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
```
Comment (Bibby0110) [inline code/modules/mob/living/simple_animal/hostile/wizard.dm:23]: atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)

File: code/modules/mob/living/simple_animal/hostile/wizard.dm [hunk 2]
```diff
@@ -18,13 +18,11 @@
...
-	unsuitable_atmos_damage = 15
```
Comment (Bibby0110) [inline code/modules/mob/living/simple_animal/hostile/wizard.dm:24]: OR
unsuitable_atmos_damage = 0

**Comment by Bibby0110** [comment]
Atmos requirements are inherited from simple_mobs

**Comment by ynot01** [comment]
downvoters are powergamers 😎

**Comment by boodaliboo** [comment]
I’ll commit those today also who the fuck cares about immersion in this game

**Comment by deathslayer2481** [comment]
Make it so when spaced wizards summon a wizard hardsuit to their body, which disappears once they die.

**Pull request merged**